frontend/src/components/collaborator/CollaboratorList.vue

```diff
@@ -35,8 +37,11 @@ export default {
   },
   computed: {
     sortedCollaborators() {
-      return [...this.collaborators].sort(
-        (a, b) => ROLE_ORDER.indexOf(a.role) - ROLE_ORDER.indexOf(b.role)
+      return sortBy(
+        [...this.collaborators],
+        (c) =>
+          ROLE_ORDER.indexOf(c.role) +
```

Comment:
This comes good as long as we have less then 10 roles.
Then, 11 will be before 2. (string sort).
but good enough for now

Reply:
Thx; I will add leading zeros (001, 002, ..., 011, 012, ...)